vars/createTestJobPipeline.groovy

```diff
@@ -0,0 +1,90 @@
+#!groovy
+
+def call() {
+
+    def builder = getJenkinsLabels("aws", "eu-west-1")
+
+    pipeline {
+        agent {
+            label {
+                   label builder.label
+            }
+        }
+        environment {
+            AWS_ACCESS_KEY_ID     = credentials('qa-aws-secret-key-id')
+            AWS_SECRET_ACCESS_KEY = credentials('qa-aws-secret-access-key')
+            SCT_TEST_ID = UUID.randomUUID().toString()
```

Comment:
JFYI: usage of single SCT_TEST_ID is not suitable for the performance tests where there are several test ids.

Reply:
Performance jobs are not created automatically. They mostly created already.
TestId is extended with datetime for each performance subtests, but main testid is same.

Reply:
Remove this line. It is not required in this job.